src/plugins/custom_integrations/public/components/fleet_integration/elasticsearch_go/elasticsearch_go_readme.tsx

```diff
@@ -0,0 +1,182 @@
+
+const CenterColumn = styled(EuiFlexItem)`
+  max-width: 740px;
+`;
+
+const FixedHeader = styled.div`
+  width: 100%;
+  height: 196px;
+  border-bottom: 1px solid ${euiThemeVars.euiColorLightShade};
+`;
+
+const IconPanel = styled(EuiPanel)`
+  padding: ${(props) => props.theme.eui.euiSizeXL};
+  width: ${(props) =>
+    parseFloat(props.theme.eui.euiSize) * 6 + parseFloat(props.theme.eui.euiSizeXL) * 2}px;
+  svg,
+  img {
+    height: ${(props) => parseFloat(props.theme.eui.euiSize) * 6}px;
+    width: ${(props) => parseFloat(props.theme.eui.euiSize) * 6}px;
+  }
+  .euiFlexItem {
+    height: ${(props) => parseFloat(props.theme.eui.euiSize) * 6}px;
+    justify-content: center;
+  }
+`;
+
+const TopFlexGroup = styled(EuiFlexGroup)`
+  max-width: 1150px;
+  margin-left: auto;
+  margin-right: auto;
+  padding: calc(${euiThemeVars.euiSizeXL} * 2) ${euiThemeVars.euiSizeM} 0 ${euiThemeVars.euiSizeM};
+`;
+
+export const ElasticsearchGoClientReadme = () => {
+  return (
+    <>
+      <FixedHeader>
+        <TopFlexGroup alignItems="center" justifyContent="flexStart">
+          <EuiFlexItem grow={false}>
+            <IconPanel>
+              <EuiImage size="fullWidth" src={icon} alt="icon" />
+            </IconPanel>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiTitle size="l">
+              <h1>
+                <FormattedMessage
+                  id="customIntegrations.languageClients.GothonElasticsearch.readme.title"
+                  defaultMessage="Elasticsearch Go Client"
+                />
+              </h1>
+            </EuiTitle>
+          </EuiFlexItem>
+        </TopFlexGroup>
+      </FixedHeader>
+
+      <EuiFlexGroup alignItems="flexStart" justifyContent="center">
+        <CenterColumn>
+          <EuiPage paddingSize="m">
+            <EuiPageBody panelled>
+              <EuiPageSection>
+                <EuiPageHeader
+                  description={
+                    <EuiText>
+                      <FormattedMessage
+                        id="customIntegrations.languageClients.GothonElasticsearch.readme.intro"
+                        defaultMessage="Getting started with the Elasticsearch Go Client requires a few steps."
+                      />
+                    </EuiText>
+                  }
+                />
+              </EuiPageSection>
+
+              <EuiPageSection>
+                <EuiTitle>
+                  <h2>
+                    <FormattedMessage
+                      id="customIntegrations.languageClients.GothonElasticsearch.readme.install"
+                      defaultMessage="Install the Elasticsearch Go Client"
+                    />
+                  </h2>
+                </EuiTitle>
+
+                <EuiSpacer size="s" />
+
+                <EuiText>
+                  <FormattedMessage
+                    id="customIntegrations.languageClients.GothonElasticsearch.readme.install7.X"
```

Comment:
Let's remove this message on installing an old version, we want users on 8.x if they have 8.x running.